internal/datanode/flow_graph_insert_buffer_node.go

```diff
@@ -98,13 +98,25 @@ func (ib *insertBuffer) size(segmentID UniqueID) int32 {
 	var maxSize int32 = 0
```

Comment:
why not change "maxSize" to int64 type ?

Reply:
@XuanYang-cn Can I change the type of maxSize to int64？